jkube-kit/common-maven/src/test/java/org/eclipse/jkube/kit/common/util/MavenUtilTest.java

```diff
@@ -102,31 +108,27 @@ public void testGetDependencies(@Mocked MavenProject mavenProject) {
     final org.apache.maven.model.Dependency dep2 = dep1.clone();
     dep2.setArtifactId("artifact2");
     dep2.setType("jar");
-    new Expectations() {
-      {
-        mavenProject.getDependencies();
-        result = Arrays.asList(dep1, dep2);
-      }
-    };
+    List<org.apache.maven.model.Dependency> dependenciesList = new ArrayList<>();
+    dependenciesList.add(dep1);
+    dependenciesList.add(dep2);
+    Mockito.when(mavenProject.getDependencies()).thenReturn(dependenciesList);
```

Comment:
1. why not using `Arrays.asList(dep1, dep2);` ?
2. try to use `import static` if possible and simplify `Mockito.when` to just `when`.

```
when(mavenProject.getDependencies()).thenReturn(Arrays.asList(dep1, dep2));
```

Reply:
1.we can do that wayy too Arrays.asList(dep1,dep2).
2.Must do thing,changing right away.